etc/picongpu/hypnos-hzdr/k80_restart.tpl

```diff
@@ -99,31 +99,32 @@ do
         file=""
     fi
 done
+export file
 
 #this sed call extracts the final simulation step from the cfg (assuming a standard cfg)
-finalStep=`echo !TBG_programParams | sed 's/.*-s[[:blank:]]\+\([0-9]\+[^\s]\).*/\1/'`
+export finalStep=`echo !TBG_programParams | sed 's/.*-s[[:blank:]]\+\([0-9]\+[^\s]\).*/\1/'`
```

Comment:
is the export of these vars and vars below really needed?

Reply:
probably not; it may have appeared there while debugging (but didn't help) and stayed there, sorry